code/lib/cli/src/automigrate/fixes/react-docgen.ts

```diff
@@ -37,15 +44,34 @@ export const reactDocgen: Fix<Options> = {
 
       https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react-docgen-component-analysis-by-default
     `;
+    } else {
+      return dedent`
+      Since Storybook 8, ${chalk.cyan(
+        'react-docgen'
+      )} is now the default for generating component controls, replacing ${chalk.cyan(
+        'react-docgen-typescript'
+      )}. 
+      This offers better performance and suits most cases. 
+      However, for complex TypeScript types or specific type features, the generated controls might not be as precise.
+      
+      For more on this change, check the migration guide: 
+      ${chalk.yellow(
+        'https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react-docgen-component-analysis-by-default'
+      )}
+      
+      For known "react-docgen" limitations, see: 
+      ${chalk.yellow('https://github.com/storybookjs/storybook/issues/26606')}
+      
+      Would you like to switch back to ${chalk.cyan('react-docgen-typescript')} in your Storybook?
```

Comment:
I think it would be nice to say something like "Your project currently uses react-docgen. Would you like to switch back to react-docgen-typescript?